CHANGELOG.md

```diff
@@ -1,5 +1,104 @@
+1. Fix problem with PYTHONNOUSERSITE being set even when not desired. There's a setting now that will set this environment variable on kernel launch if it's needed: 'jupyter.excludeUserSitePackages'.
+   ([#9995](https://github.com/Microsoft/vscode-jupyter/issues/9995))
+1. Don't show the python extension install ui when auto starting kernels.
+   ([#10011](https://github.com/Microsoft/vscode-jupyter/issues/10011))
+1. Support standard ipywidgets in the web extension.
+   ([#10051](https://github.com/Microsoft/vscode-jupyter/issues/10051))
+1. When connecting to a remote Jupyter server with a password you will not have to input the server URL twice to have it apply anymore.
+   ([#10103](https://github.com/Microsoft/vscode-jupyter/issues/10103))
+1. Fix clicking on links for error callstacks to open the same original python file instead of a new one.
+   ([#10149](https://github.com/Microsoft/vscode-jupyter/issues/10149))
+1. Gracefully handle failures when attempting to convert ANSI codes to HTML in large error output within the errors renderer.
+   ([#10172](https://github.com/Microsoft/vscode-jupyter/issues/10172))
+1. Fix 'go to source' to work again in the interactive window.
+   ([#10205](https://github.com/Microsoft/vscode-jupyter/issues/10205))
+1. Fixes run by line not stopping on any lines.
+   ([#10207](https://github.com/Microsoft/vscode-jupyter/issues/10207))
+1. Fix Jupyter: Variables hiding when closing a notebook or an interactive window.
+   ([#10209](https://github.com/Microsoft/vscode-jupyter/issues/10209))
+1. Fix error renderer to return tracebacks.
+   ([#10239](https://github.com/Microsoft/vscode-jupyter/issues/10239))
+1. Fix problem with interactive window links being off by one.
+   ([#10283](https://github.com/Microsoft/vscode-jupyter/issues/10283))
+
+### Code Health
+
+1. Add test to make sure if the active interpreter is switched, the interactive window switches to that interpreter.
+   ([#5478](https://github.com/Microsoft/vscode-jupyter/issues/5478))
+1. Add tests to verify notebook metadata is in a notebook.
+   ([#5601](https://github.com/Microsoft/vscode-jupyter/issues/5601))
+1. Add telemetry test to verify we output a specific set for different operations.
+   ([#6883](https://github.com/Microsoft/vscode-jupyter/issues/6883))
+1. Added a Performance test to test and ensure expected performance characteristics.
+   ([#7437](https://github.com/Microsoft/vscode-jupyter/issues/7437))
+1. Don't skip code completions when the server is busy. Instead let it timeout if the server doesn't come back.
+   ([#9797](https://github.com/Microsoft/vscode-jupyter/issues/9797))
+1. Add test for remote https jupyter servers.
+   ([#9844](https://github.com/Microsoft/vscode-jupyter/issues/9844))
+1. Create test for turning on old debugger tests and create a test for deleting tmp test files.
+   ([#10041](https://github.com/Microsoft/vscode-jupyter/issues/10041))
```

Comment:
This issue isn't closed, and it has a regression it seems.

Reply:
I'll remove it from the changelog.